Tests/sdk_release/create_sdk_pr.py

```diff
@@ -177,6 +177,7 @@ def main():
     ).get('id')
 
     logging.info(f'SDK changelog workflow triggered successfully: {SDK_WORKFLOW_SUFFIX}{workflow_id}')
+    time.sleep(10)
```

Comment:
@adi88d can't we poll instead of sleep in some way? if not that's fine.

Reply:
i moved the sleep to the beginning of the while